hepcrawl/spiders/elsevier_spider.py

```diff
@@ -203,9 +204,15 @@ def unzip_zip_package_to_s3(self, response):
                         callback=self.parse_record,
                     )
 
+    def _file_name_from_url(self, url):
+        return basename(urlparse(url).path)
+
     def parse_record(self, response):
         """Parse an elsevier XML downloaded from s3 into a HEP record."""
         parser = ElsevierParser(response.meta['data'])
-
-        return ParsedItem(record=parser.parse(), record_format="hep")
+        file_name = self._file_name_from_url(response.url)
+        parser.attach_fulltext_document(file_name, response.url)
```

Comment:
not sure it's happening, but the attach fulltext should be hidden

Reply:
Yes it is. Here `attach_fulltext_document` is used and as you can se in  it's [definition](https://github.com/inspirehep/hepcrawl/blob/52db9383d077c24b726137128428abff0a20af38/hepcrawl/parsers/elsevier.py#L606-L607) it's adding file as hidden.